### ISSUE TITLE
GPU memory

### ISSUE THREAD

**Issue by futianfan** [lmthang/nmt.hybrid#1]
Hi,

Nice work!
I want to re-train the deep and large model described in the paper. Could you please tell me how many GPUs or how much GPU memory do I need? It seems that a single Titan X GPU is not enough.

Thanks!

**Comment by lmthang** [comment]
Hi futianfan,

I used only 1 GPU (Tesla K40) and it can fit in memory just fine, about 6-8gb for the deep hybrid models if I recall correctly. Matlab, while not as nice as framework like TensorFlow, allows me to customize many things ;)

-Thang

**Comment by futianfan** [comment]
Thanks! It is an advantage of using Matlab,  I will try to re-train the model.
